sdk/python/kfp_tekton/compiler/_data_passing_rewriter.py

```diff
@@ -343,54 +291,55 @@ def mark_upstream_ios_of_output(template_output, marked_inputs,
         for output in outputs_consumed_as_artifacts
     }
     # task_workspaces = set()
-    for pipeline in pipeline_templates:
-        # Converting pipeline inputs
-        pipeline, pipeline_workspaces = big_data_passing_pipeline(
-            pipeline, inputs_consumed_as_artifacts,
-            output_tasks_consumed_as_artifacts)
+
+    # Converting pipeline inputs
+    pipeline, pipeline_workspaces = big_data_passing_pipeline(
+        pipelinerun_name, pipeline_template, inputs_consumed_as_artifacts,
+        output_tasks_consumed_as_artifacts)
 
     # Add workspaces to pipelinerun if big data passing
-    # Check whether pipelinerun was generated, through error if not.
+    # Check whether pipelinerun was generated, throw error if not.
```

Comment:
remove this comment, we always generate a PipelineRun now :-)

Reply:
Addressed